src/devtools/client/webconsole/reducers/messages.ts

```diff
@@ -259,12 +265,6 @@ const messagesSlice = createSlice({
 
       return removeMessagesFromState(state as MessageState, removedIds);
     },
-    //  This is only here to force recalculation after filters are updated
```

Comment:
🎉

Reply:
Yeah, happened to notice that this wasn't being referenced anywhere so ✂️  😁